src/core/components/footer/styles.ts

```diff
@@ -18,11 +18,28 @@ export const footer = ({
 	}
 `
 
+const getMarginBottom = (initialSpace: number, withBackToTop: boolean) => {
+	if (!withBackToTop) {
+		return initialSpace;
+	}
+
+	return initialSpace - height.ctaMedium / 2
+}
+export const linksWrapper = (withBackToTop: boolean) => css`
+	display: flex;
+	align-items: center;
+	margin-bottom: ${getMarginBottom(space[6], withBackToTop)}px;
+
+	${from.desktop} {
+		margin-bottom: ${getMarginBottom(space[1], withBackToTop)}px;
+	}
+`
```

Comment:
This is looking pretty good, but I'd prefer if we could do this statically rather than using dynamic styles. Rather than using a function here, could we split this up into separate styles and move the logic that determines which styles to apply into the component?

e.g.

```tsx
export const linksWrapper = css`
	display: flex;
	align-items: center;
`

export const linksWrapperSpace = css`
    margin-bottom: ${space[6]}px;
	${from.desktop} {
		margin-bottom: ${space[1]}px;
	}
`

const backToTopSpace = (initial) => initial - height.ctaMedium / 2

export const linksWrapperSpaceWithBackToTop = css`
    margin-bottom: ${backToTopSpace(space[6]}px;
	${from.desktop} {
		margin-bottom: ${backToTopSpace(space[1]}px;
	}
`
```

Let me know if that makes sense